Sources/SourceKitLSP/Swift/SemanticTokens.swift

```diff
@@ -107,7 +110,10 @@ extension SyntaxClassifiedRange {
       return SyntaxHighlightingTokens(tokens: [])
     }
 
-    let multiLineRange = snapshot.positionOf(utf8Offset: self.offset)..<snapshot.positionOf(utf8Offset: self.endOffset)
+    let multiLineRange =
+      snapshot.positionOf(
+        utf8Offset: range.lowerBound.utf8Offset
+      )..<snapshot.positionOf(utf8Offset: range.upperBound.utf8Offset)
```

Comment:
This can be simplified slightly

```suggestion
    let multiLineRange = snapshot.range(of: self.range)
```